com.unity.render-pipelines.universal/ShaderLibrary/Shadows.hlsl

```diff
@@ -341,7 +342,11 @@ half AdditionalLightShadow(int lightIndex, float3 positionWS, half4 shadowMask,
 
 float4 GetShadowCoord(VertexPositionInputs vertexInput)
 {
+#ifdef _MAIN_LIGHT_SHADOWS_SCREEN
```

Comment:
Change to `#if defined(_MAIN_LIGHT_SHADOWS_SCREEN)` for consistency

Reply:
Ok, I will. thanks :)